Sources/SpeziLLM/Helpers/LLMContext+Chat.swift

```diff
@@ -6,7 +6,7 @@
 // SPDX-License-Identifier: MIT
 //
 
-import SpeziChat
+@preconcurrency import SpeziChat
```

Comment:
Makes sense for now, but we should also lift `SpeziChat` (and the depending `SpeziSpeech`) to Swift 6 language mode.
As this SpeziLLM PR will take a while to get merged (because of the OpenAI layer rewamp), it might make sense to start with `SpeziChat` and `SpeziSpeech` in the meantime.
See the issue here: https://github.com/StanfordSpezi/SpeziChat/issues/15

Reply:
Tackled in https://github.com/StanfordSpezi/SpeziSpeech/pull/10 and https://github.com/StanfordSpezi/SpeziChat/pull/20